src/common/ActionButtons/ActionButton.tsx

```diff
@@ -0,0 +1,85 @@
+import { FC } from "react";
+import classnames from "classnames";
+
+import { ActionButtonsProps } from "./ActionButtons";
+
+import { useMessageContext } from "../../hooks";
+import { getWebchatButtonLabel } from "../../utils";
+import { sanitizeHTML } from "../../sanitize";
+
+import classes from "./ActionButton.module.css";
+
+interface ActionButtonProps extends React.HTMLAttributes<HTMLDivElement> {
+	action: ActionButtonsProps["action"];
+	className?: string;
+	button: ActionButtonsProps["payload"][number];
+	total: number;
+	position: number;
+	disabled?: boolean;
+}
+
+/**
+ * Postback, phone number, and URL buttons
+ */
+const ActionButton: FC<ActionButtonProps> = props => {
+	const { button, total, position } = props;
+	const { config, onEmitAnalytics } = useMessageContext();
+
+	const buttonLabel = getWebchatButtonLabel(button) || "";
+	const __html = config?.settings?.disableHtmlContentSanitization
+		? buttonLabel
+		: sanitizeHTML(buttonLabel);
```

Comment:
on the old code (line 139 on file WebchatWidget/src/plugins/messenger/index.tsx), we have a condition that you skipped here. It was intentional?

Reply:
good catch, fixed